edweiss-app/package-lock.json

```diff
@@ -5502,9 +5502,9 @@
       }
     },
     "node_modules/@react-native-async-storage/async-storage": {
-      "version": "1.23.1",
-      "resolved": "https://registry.npmjs.org/@react-native-async-storage/async-storage/-/async-storage-1.23.1.tgz",
-      "integrity": "sha512-Qd2kQ3yi6Y3+AcUlrHxSLlnBvpdCEMVGFlVBneVOjaFaPU61g1huc38g339ysXspwY1QZA2aNhrk/KlHGO+ewA==",
+      "version": "1.24.0",
```

Comment:
I can delete this file if you thinks that is better.

Reply:
I don't know better check with more experienced people.

Reply:
Hello, I think we should leave the version of async storage as it is because it has been running. Can you check if having the previous version, the tests still run and there isn't any error?